docs/rebuild_admin.rst

```diff
@@ -172,8 +167,9 @@ and deleting the line:
 
 Then, restart ``sshd`` using the command ``sudo service sshd restart``.
 
-Finally, check the file ``/etc/network/interfaces``, and note down the network
-settings for the default Ethernet interface. You'll need them in the next step.
+Finally, check the file ``/etc/netplan/00-installer-config.yaml``, and note
```

Comment:
As far as I can tell, all we need in the next step is the IP address of both servers. Is there a reason not to use the simpler `ip addr` (`ip a`) command here, particularly considering that the location of this file might change in the course of administration?